app/models/service_template.rb

```diff
@@ -509,7 +513,10 @@ def resource_action_list
 
   def update_from_options(params)
     options[:config_info] = params[:config_info]
-    update_attributes!(params.except(:config_info))
+    if params[:picture]
+      update_attributes!(:picture => Picture.create(:content => params[:picture][:content], :extension => 'jpg'))
```

Comment:
Do we only support jpg?  If not, how do we know the extension?

Reply:
ugh, fine, no, to the first: https://github.com/ManageIQ/manageiq/blob/master/app/models/picture.rb#L5 and ugh, fine, i guess it's we assume it's in the params too? maaaaaybe?

Reply:
Maybe?  I don't know how we create pictures.